docker-compose.yml

```diff
@@ -0,0 +1,72 @@
+# Temporary docker compose file to spin up the services needed to support bio auth
+# TODO eventually this will all get moved over to the guardian-bio-auth repo
+#
+version: '3.5'
+
+networks:
+  internal-network:
+    name: internal-network
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+        - subnet: 10.0.0.0/24
+
+services:
+  identity-service:
+    build: ./identity_service
+    image: identity-service
+    container_name: identity-service
+    ports:
+      - "8081:8080"
+    env_file:
+      - identity_service/.env
+    networks:
+      - internal-network
+    depends_on:
+      - identity-template-db
+      - identity-intelligence-db
+    tty: true
+
+  identity-template-db:
+    command: postgres -c logging_collector=on -c log_destination=stderr -c log_directory=/logs
```

Comment:
We discovered postgress is a bit of a problem on windows.  Since we are looking to open source this, might want to consider the change we made in aries-guardianship-agency so that it will work on windows. https://github.com/kiva/aries-guardianship-agency/blob/775432c81e70f70939c93b1cb7536802978d9714/docker-compose.yml#L43

I am fine if we do/or dont.  Sharing what we learned.

Reply:
That's a good call. Also, I think Windows will have an issue with `/logs` vs `\logs`.